Tests/ProGetAutomationTest/ProGetAutomationTest.psm1

```diff
@@ -9,15 +9,50 @@ if( -not $svcRoot )
     throw $pgNotInstalledMsg
 }
 
-$svcConfig = [xml](Get-Content -Path (Join-Path -Path $svcRoot -ChildPath 'ProGet.Service.exe.config' -Resolve) -Raw)
-if( -not $svcConfig )
+$uri = ('http://{0}:82/' -f [Environment]::MachineName)
+
+$configFiles = & {
+                    Join-Path -Path $svcRoot -ChildPath 'ProGet.Service.exe.config'
+                    Join-Path -Path $svcRoot -ChildPath 'App_appsettings.config'
+                    Join-Path -Path $env:ProgramData -ChildPath 'Inedo\SharedConfig\ProGet.config'
+                } | 
+                Where-Object { Test-Path -Path $_ -PathType Leaf }
+
+foreach( $configPath in $configFiles )
 {
-    throw $pgNotInstalledMsg
+    $configContent = Get-Content -Raw -Path $configPath 
+    $configContent | Write-Debug
+    $svcConfig = [xml]$configContent
+    if( -not $svcConfig )
+    {
+        throw $pgNotInstalledMsg

```

Comment:
Note for posterity: There is a [Docker image for ProGet](https://hub.docker.com/r/inedo/proget) which would be neat to integrate into this repo for simpler test instance setup.